samples/python/com.ibm.streamsx.topology.pysamples/opt/python/streams/spl_samples.py

```diff
@@ -0,0 +1,140 @@
+# Licensed Materials - Property of IBM
+# Copyright IBM Corp. 2015
+
+# Simple inclusion of Python logic within an SPL application
+# as a SPL "Function" operator. A "Function" operator has
+# a single input port and single output port, a function
+# is called for every input tuple, and results in
+# no submission or a single tuple being submitted.
```

Comment:
namespace-info.spl indicates that 0,1, or more tuples may be generated for each input tuple.  
namespace-info.spl/124 indicates @spl.sink functional ops do not have an oport.  (applies to line 15 below too)